media-gfx/qview/qview-3.0.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="Minimalist image viewer"
+DESCRIPTION="qView is an image viewer designed with minimalism and usability in mind."
+HOMEPAGE="https://interversehq.com/qview/"
+COPYRIGHT="2018-2020 jurplel and qView contributors"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/jurplel/qView/archive/3.0.zip"
+CHECKSUM_SHA256="5bc2a2eb3aa6329fe6f207a72497f115625d84a0e76663872fd2e401ea7c5ae1"
+# Este parametro es para que pueda encontrar la carpeta de fuentes luego de desempaquetar el zip:
+SOURCE_DIR="qView-$portVersion"
+#PATCHES=""
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	qview$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:make
+	cmd:qmake$secondaryArchSuffix >= 5
+	"
+
+BUILD()
+{
+	qmake .
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir $appsDir
```

Comment:
Always use the -p option wit mkdir. Otherwise it'll fail if the directory already exists.
```suggestion
	mkdir -p $appsDir
```